samples/symIO.c

```diff
@@ -0,0 +1,170 @@
+         puts( "Press  ENTER without any command to see register  values" );
+         puts( "without changing any of them.   Type 'help' to see these" );
+         puts( "instructions again and type 'stop' to end the program.\n");
+         puts( "Available registers:   DDR1A, IOR1A, DDR1B, IOR1B, DDR2A" );
+         puts( "IOR2A, DDR2B, IOR2B, DDR3A, IOR3A, DDR3B and IOR3B." );
+         instr = 0;
+      }
+
+      printf( "\n Command:  " );
+
+      fgets(cmd, sizeof(cmd)-1, stdin);
+      cmd[strlen(cmd)-1] = '\0';
+
+      if( strncasecmp(cmd, "stop", 4) == 0) {
+         going = 0;  
+      }
+      else if( strncasecmp(cmd, "help", 4) == 0) {
+         instr = 1;
+      }
+      else if( strncasecmp(cmd, "ddr1a", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = atoi( vp );
+            set_DDR1A( val );
+         }
+      }
+      else if( strncasecmp(cmd, "ior1a", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = atoi( vp );
+            set_IOR1A( val );
+         }
+      }
+      else if( strncasecmp(cmd, "ddr1b", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = atoi( vp );
+            set_DDR1B( val );
+         }
+      }
+      else if( strncasecmp(cmd, "ior1b", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = atoi( vp );
+            set_IOR1B( val );
+         }
+      }
+      else if( strncasecmp(cmd, "ddr2a", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = atoi( vp );
+            set_DDR2A( val );
+         }
+      }
+      else if( strncasecmp(cmd, "ior2a", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = atoi( vp );
+            set_IOR2A( val );
+         }
+      }
+      else if( strncasecmp(cmd, "ddr2b", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = atoi( vp );
+            set_DDR2B( val );
+         }
+      }
+      else if( strncasecmp(cmd, "ior2b", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = atoi( vp );
+            set_IOR2B( val );
+         }
+      }
+      else if( strncasecmp(cmd, "ddr3a", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = atoi( vp );
+            set_DDR3A( val );
+         }
+      }
+      else if( strncasecmp(cmd, "ior3a", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = atoi( vp );
+            set_IOR3A( val );
+         }
+      }
+      else if( strncasecmp(cmd, "ddr3b", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = atoi( vp );
+            set_DDR3B( val );
+         }
+      }
+      else if( strncasecmp(cmd, "ior3b", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = atoi( vp );
+            set_IOR3B( val );
+         }
+      }
+   }
+
+   return;
+}
```

Comment:
Please change this code to adhere to the cc65 C code style rules - see https://github.com/cc65/wiki/wiki/C-and-Assembly-sources-style-guide

Reply:
**DONE**.  Please check and confirm.